packages/alfa-rules/test/sia-r54/rule.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import { test } from "@siteimprove/alfa-test";
+
+import { Document } from "@siteimprove/alfa-dom";
+
+import R54, { Outcomes } from "../../src/sia-r54/rule";
+
+import { evaluate } from "../common/evaluate";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+test("evaluate() passes <div> element with aria-live and aria-atomic ", async (t) => {
+  const target = (
+    <div aria-live="assertive" aria-atomic="true">
+      Some words
+    </div>
+  );
+
+  const document = Document.of([target]);
```

Comment:
We should be using `h.document` instead of `Document.of`…
See https://github.com/Siteimprove/alfa/pull/771#discussion_r649872631 and https://github.com/Siteimprove/alfa/commit/ce61ed613795fbe131cffa8a462fffd141efa171